base/applications/cmdutils/tasklist/lang/bg-BG.rc

```diff
@@ -0,0 +1,20 @@
+LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE, "Usage: TASKLIST [/?] [/V]\n"
+    IDS_DESCRIPTION, "\n\
+Desciption:\
+    This tool displays a list of currently running processes on\n\
+    the machine.\n"
+    IDS_PARAM_TOO_MUCH, "Error: Invalid syntax. '%1' option is not allowed more than '%2!d!' time(s).\n"
+    IDS_INVALID_SYNTAX, "Error: Invalid syntax.\n"
+    IDS_INVALID_OPTION, "Error: Unknown or invalid command line option specified.\n"
+    IDS_ENUM_FAILED, "Error: Unable to enumerate the process list.\n"
+    IDS_OUT_OF_MEMORY, "Error: Out of memory.\n"
+    IDS_HEADER_IMAGENAME, "Image Name"
+    IDS_HEADER_PID, "PID"
+    IDS_HEADER_SESSION, "Session#"
+    IDS_HEADER_MEMUSAGE, "Mem Usage"
+END
+
```

Comment:
What's with the redundant whitespace here?

Reply:
removed now!